examples/neighbors/plot_nca_illustration.py

```diff
@@ -57,8 +56,7 @@ def relate_point(X, i, ax):
         thickness = p_i(X, i)
         if i != j:
             line = ([pt_i[0], pt_j[0]], [pt_i[1], pt_j[1]])
-            ax.plot(*line, c=cm.Set1(y[j]),
-                    linewidth=5*thickness[j])
+            ax.plot(*line, c=cm.Set1(y[j]), linewidth=5*thickness[j])
```

Comment:
This doesn't use np.array. Why is the change not needed here?

Reply:
For the y array it accepts without any warnings but for the x one it refuses

Reply:
You don't need this change (it is only in `scatter` plot)